app/code/core/Mage/ImportExport/Model/Resource/Helper/Mysql4.php

```diff
@@ -54,12 +54,17 @@ public function getMaxDataSize()
      */
     public function getNextAutoincrement($tableName)
     {
-        $adapter = $this->_getReadAdapter();
-        $entityStatus = $adapter->showTableStatus($tableName);
+        $connection = $this->_getReadAdapter();
 
-        if (empty($entityStatus['Auto_increment'])) {
+        $sql = sprintf(
+            'SELECT AUTO_INCREMENT FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_NAME = %s AND TABLE_SCHEMA = DATABASE()',
+            $this->quote($tableName)
```

Comment:
Should this be like so?
```suggestion
            $connection->quote($tableName)
```